demo/sm_42v_1.tscn

```diff
@@ -1,7 +1,7 @@
 [gd_scene format=3 uid="uid://c5hi8nsm1d2hb"]
 
 [node name="SM42v1" type="TrainController"]
-parts/engine = NodePath("StonkaDieselEngine")
+parts/engine = NodePath("TestowySilnikElektryczny")
```

Comment:
silnik stonki zmienił się na elektryka, domyślnie zostawiłbym diesla tak jak było

Reply:
Zapomniałem tho xD

Reply:
Będzie nam to wracało, to póki będzie jeden lok z dwoma silnikami ;)